setup.cfg

```diff
@@ -1,3 +1,15 @@
+[isort]
```

Comment:
What I think should happen, if you want to suggest sorted imports, is a new PR:

1. Title should be "Require imports to be sorted in style rules"
2. It should implement a flake8 check that rejects PRs that have unsorted imports
3. Optionally, it can provide a tool that automatically sorts, but that tool sould work by default on a file-by-file basis. We shouldn't default to altering the whole codebase.
4. The sorting tool should be documented somewhere.

Once that PR has been agreed and merged, we can push a follow up PR that does an initial "get everything sorted" change. Ideally, we'd scan existing PRs to ensure we don't break too many, but that's probably too hard to be worthwhile.